runtime/compiler/control/JitDump.cpp

```diff
@@ -586,15 +586,29 @@ runJitdump(char *label, J9RASdumpContext *context, J9RASdumpAgent *agent)
                if (!(jittedMethodsOnStack[i]._method))
                   continue;
 
+               bool isAOTBody = false;
+               J9JITExceptionTable *metadata = NULL;
+               void *startPC = jittedMethodsOnStack[i]._oldStartPC;
+               if (startPC)
+                  {
+                  metadata = jitConfig->jitGetExceptionTableFromPC(crashedThread, (UDATA)startPC);
+                  if (metadata)
+                     {
+                     TR_PersistentJittedBodyInfo *bodyInfo = (TR_PersistentJittedBodyInfo *)metadata->bodyInfo;
+                     if (bodyInfo)
+                        isAOTBody = bodyInfo->getIsAotedBody();
+                     }
+                  }
+
                TR_CompilationErrorCode compErrCode;
                compErrCode = recompileMethodForLog(
                   crashedThread,
                   jittedMethodsOnStack[i]._method,
                   compInfo,
-                  frontendOfThread,
                   jittedMethodsOnStack[i]._optLevel,
                   false,
-                  jittedMethodsOnStack[i]._oldStartPC,
+                  isAOTBody,
```

Comment:
I should note, I don't think this will actually result in an AOT compilation; this is because the method is already in the SCC, so the control code will probably just try to do an AOT load. However, if the AOT compilation was also done in the current JVM, it would be nice to be able to log that compilation as well, though this would require keeping track of the methods that were AOT compiled by the current JVM.

Reply:
Will you add more code to bypass those AOT loads? Otherwise the current change does not achieve its intended effect.

Reply:
I figured I would leave that for a future PR, since trying to do that in this PR might result in quite a big PR.